### PR TITLE
Update the jsdoc comments to modern syntax - Part 4 

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
`src/js/tech/*`
`src/js/poster-image`
## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/js/tech/html5.js [hunk 1]
```diff
@@ -833,64 +947,83 @@ Html5.Events = [
 ];
 
 /**
- * Set the tech's volume control support status
+ * Boolean indicating wether the `Tech` supports volume control.
```
Comment (misteroneill) [inline src/js/tech/html5.js]: Should be "whether" - this is found in a few places.

File: src/js/tech/html5.js [hunk 2]
```diff
@@ -1004,172 +1137,304 @@ Html5.resetMediaElement = function(el) {
 // Wrap native properties with a getter
 [
   /**
-   * Paused for html5 tech
+   * Get the value of `paused` from the media element. `paused` indicates wether the media element
+   * is current paused or not.
```
Comment (misteroneill) [inline src/js/tech/html5.js]: Looks like this should be "currently".

File: src/js/tech/html5.js [hunk 3]
```diff
@@ -1004,172 +1137,304 @@ Html5.resetMediaElement = function(el) {
...
    */
   'currentTime',
+
   /**
-   * Get a TimeRange object that represents the intersection
-   * of the time ranges for which the user agent has all
-   * relevant media
+   * Get the value of `buffered` from the media element. `buffered` is a `TimeRange`
+   * object that represents the parts of the media that are already downloaded and
+   * avialable for playback.
```
Comment (misteroneill) [inline src/js/tech/html5.js]: Typo `avialable`

File: src/js/tech/html5.js [hunk 4]
```diff
@@ -1004,172 +1137,304 @@ Html5.resetMediaElement = function(el) {
...
+   *
+   * @see [Spec]{@link https://www.w3.org/TR/html5/embedded-content-0.html#dom-media-buffered}
    */
   'buffered',
+
   /**
-   * Get volume level
+   * Get the value of `volume` from the media element. `volume` indicates
+   * the current playback volume of audio for a media. `volume` will be a value from 0
+   * (silent) to 1 (loudest, and default).
```
Comment (misteroneill) [inline src/js/tech/html5.js]: Comma isn't necessary here.

File: src/js/tech/tech.js [hunk 5]
```diff
@@ -20,6 +18,48 @@ import MediaError from '../media-error.js';
 import window from 'global/window';
 import document from 'global/document';
 
+/**
+ * An Object containing a structure like: `{src: 'url', type: 'mimetype'}` or  string
+ * that just contains a only the src url.
```
Comment (misteroneill) [inline src/js/tech/tech.js]: Looks like "a only" should be "only"

File: src/js/tech/tech.js [hunk 6]
```diff
@@ -126,18 +171,32 @@ class Tech extends Component {
   }
 
   /**
-   * Track progress
+   * This is used to trigger a `progress` event when the buffered percent changes. It
+   * sets an interval function that will be called every 500miliseconds to check if the
```
Comment (misteroneill) [inline src/js/tech/tech.js]: Should be "500 milliseconds"

File: src/js/tech/tech.js [hunk 7]
```diff
@@ -126,18 +171,32 @@ class Tech extends Component {
...
+   * bufferend percent has changed.
```
Comment (misteroneill) [inline src/js/tech/tech.js]: Typo "bufferend"

File: src/js/tech/tech.js [hunk 8]
```diff
@@ -263,15 +338,14 @@ class Tech extends Component {
   }
 
   /**
-   * clear out a track list, or multiple track lists
+   * Clear out a single `TrackList`, or an array of  `TrackLists`.
```
Comment (misteroneill) [inline src/js/tech/tech.js]: Don't need a comma here.

File: src/js/tech/tech.js [hunk 9]
```diff
@@ -292,20 +366,20 @@ class Tech extends Component {
...
-   * @param {MediaError=} err    Optional an error object
-   * @return {MediaError}        the current error object or null
-   * @method error
+   * Get or set an error on the Tech.
+   *
+   * @param {MediaError} [err]
+   *        Error to set on the Tech
+   *
+   * @return {MediaError|null}
+   *         The curren error object on the tech, or null if there isn't one.
```
Comment (misteroneill) [inline src/js/tech/tech.js]: Typo "curren"

File: src/js/tech/tech.js [hunk 10]
```diff
@@ -332,24 +407,39 @@ class Tech extends Component {
   }
 
   /**
-   * Set current time
+   * Causes a manual time update to occure if {@link Tech#manualTimeUpdatesOn} was
```
Comment (misteroneill) [inline src/js/tech/tech.js]: Typo "occure"

File: src/js/tech/tech.js [hunk 11]
```diff
@@ -529,10 +649,20 @@ class Tech extends Component {
   /**
    * Creates a remote text track object and returns a emulated html track element
    *
-   * @param {Object} options The object should contain values for
-   * kind, language, label and src (location of the WebVTT file)
+   * @param {Object} options
+   *        The object should contain the options to intialize the TextTrack with.
```
Comment (misteroneill) [inline src/js/tech/tech.js]: Typo "intialize"

**Comment by BrandonOCasey** [comment]
We may just want to find some tool to do grammar checks in another PR

**Comment by BrandonOCasey** [comment]
maybe a combination of https://www.npmjs.com/package/eslint-plugin-spellcheck and https://www.npmjs.com/package/write-good

File: src/js/tech/tech.js [hunk 12]
```diff
@@ -338,7 +338,7 @@ class Tech extends Component {
   }
 
   /**
-   * Clear out a single `TrackList`, or an array of  `TrackLists`.
+   * Clear out a single `TrackList` or an array of `TrackLists` givent their names.
```
Comment (BrandonOCasey) [inline src/js/tech/tech.js]: nice catch!

**Pull request merged**